server/reports/encounters/2_6_0/src/template.html

```diff
@@ -0,0 +1,49 @@
+{% macro formatDate(datetime) %} {% if datetime %}{{ datetime |
+date(format="%d/%m/%Y") }}{% endif %} {% endmacro input %}
+
+<style>
+  {% include "encounters.css" %}
```

Comment:
I haven't checked the other reports' css, but I assume there should be some pretty common css that can be shared amongst reports? That said perhaps there would be some nastiness with doing that with the report versioning.

Reply:
Yeah I think I remember it being a versioning thing... though something shared would be nice 🥲